DOTCHI/DOTCHI/Global/Bases/BaseViewController.swift

```diff
@@ -180,6 +180,34 @@ OS Version: \(UIDevice.current.systemVersion)
         }
     }
     
+    func sendMail() {
```

Comment:
위에 비슷한 역할의 함수가 `sendForgetPasswordMail()` 이었는데요. 이것도 무슨 메일인지 목적이 정확하면 좋을 것 같아요~
```suggestion
    func sendContactMail() {
```
등..